### ISSUE TITLE
Update Accessibility Conformance Documentation

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#11272]
GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11yTCS;#A11yMAS;#A11yAAT;#DesktopApp;#SQL Azure Data Studio;#A11ySev3;#Win32;

**Recommendation**: "Fix and close other bugs before starting accessibility conformance documentation.  Any bug that will not be fixed and closed within SLA must be declared in the AAT.  The SLA for this bug is also 60 days."

**Link to the current AAT entry**:-https://cela-ra-aat-prod.azurewebsites.net/#/workflow/ee5c9007-47f4-4fcb-a4d7-a998a617edde

 **Update and Republish conformance documentation**

When the AAT entry is completed, please resolve the bug with the AAT URL.  Trusted tester will review the AAT and send comments if any.  Once this review is completed, then mark the AAT entry for review in the AAT itself - this will send the entry to CELA for their review.  When CELA publishes the document, the bug will be closed.

**France Reporting Reqs**: please refer to  "French Decree Scope for Websites" page  to determine if the product is in scope for French decree or not.  If it is in scope, verify that the French Reporting Reqs section of the AAT is filled in (for questions on what and how to verify content in this section, please contact Chitra J.).  If it is NOT in scope, then the checkbox in the AAT that reads "To the best of my knowledge, this website, service or application does not have French reporting obligations." must be checked.

**Comment by hannahqin** [comment]
Closing as duplicate
